tests/test_data.py

```diff
@@ -557,6 +557,16 @@ def test_modeledparameters(db):
     t = db.query(db.ModeledParameters).filter(db.ModeledParameters.c.reference == ref).astropy()
     assert len(t) == 696, f'found {len(t)} modeled parameters with {ref} reference'
 
+    #Test to verify T eff counts
+    ref = 'T eff'
```

Comment:
```suggestion
    parameter = 'T eff'
```
Let's rename this variable just to be more accurate.